docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -68,15 +39,7 @@ parameters:
 
 [horizontal]
 type:: dictionary
-default::
-+
-[source,yaml]
-----
-ssd-local:
-  class: ssd
-  fstype: ext4
-  volumeexpansion: true
-----
+default:: {}
```

Comment:
```suggestion
default:: []
```

Reply:
Probably an example like in https://github.com/topolvm/topolvm/blob/main/charts/topolvm/values.yaml#L430 could make sense.